main/EventSystem.cpp

```diff
@@ -2442,27 +2440,28 @@ void CEventSystem::ExportDomoticzDataToLua(lua_State *lua_state, uint64_t device
 			_hardwareNames[ID] = tlist;
 		}
 	}
+	
+	Json::Value tempjson;
+	m_webservers.GetJSonDevices(tempjson, "true", "", "dzvID", "", "", "", true, false, false, 0, "");
```

Comment:
The json has more (constructed) information (formatted) that is unfortunately not in the database otherwise we would definitely have used it.  In EventSystem.cpp at the top you see the list (_tJsonLuaMap) of items that are only available through the json data structure. There is quite some logic in that json-generation code specifically for this data preparation (previously for the GUI alone but dzVents needs that info too). 
But agreed, it's a very expensive call.

@scottydj2 / @jvandenbroek  correct me if I'm wrong.